### DISCUSSION TITLE
3.10.6-->3.11.2???

### DISCUSSION THREAD

**Discussion by boxallw** [OnionUI/Onion#516] [Q&A] [answered]
How do I make that happen? Can't find any instructions.

**Comment by Sichroteph** [comment, marked as answer]
Follow the upgrade instructions here 
https://github.com/jimgraygit/Onion#installation-upgrade

**Reply by boxallw** [reply]
Upgrade says....

<= V3.09

I am 3.10.6 so not <= 3.09

**Reply by Olywa** [reply]
It says "(If coming from <= V3.09, backup your saves (retroarch / .retroarch / saves | states), you will have to copy them afterwards in the Saves folder of your SD)"

Basically if you were coming from that version or before then you would need to move your saves but since you are already on V3.10.6 you can ignore this line as you will already have a 'Saves' folder on your SD root (that contains your saves/states and RA configs). Just follow rest of the upgrade instructions after this point.
